### PR TITLE
add browser extension package

### PR DESCRIPTION
**Pull request by Evanfeenstra**
# Description of change

Add browser extension package into Firefly

## Type of change

Choose a type of change, and delete any options that are not relevant.

- New (a change which implements a new feature)

## Change checklist

- [ x ] I have performed a self-review of my own code
- [ x ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by maxwellmattryan — APPROVED** [review]
LGTM! Thanks :pray:

File: packages/extension/web/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "glow",
```
Comment (maxwellmattryan) [inline packages/extension/web/package.json:2]: To match the manifest, we should also name this to "firefly-web" or whatever you think is best.

**Review by kraftjs — APPROVED** [review]
Looks good to me 👍🏻

File: packages/extension/web/package.json (hunk 1 above)
Comment (Evanfeenstra) [reply packages/extension/web/package.json]: ok, renamed to `firefly-extension`

Comment (maxwellmattryan) [reply packages/extension/web/package.json]: Thank you :pray:

**Comment by begonaalvarezd** [comment]
Unfortunately this feature has been paused for now.
Thank you very much for the work 🌷

**Pull request closed without merging**